### ISSUE TITLE
Get to Axe Clean

### ISSUE THREAD

**Issue by TheBryceIsRight** [TheBryceIsRight/system-status#18]
Website should be free of any accessibility errors. Currently there are 6 that are moderate in severity, though 90% of them are related to the top bar. 

![axe readout.PNG](https://images.zenhubusercontent.com/592dcd7151aa2c30577fe56b/27b7c14a-5c35-42aa-92e6-3a01317c897b)

**Comment by TheBryceIsRight** [comment]
The main index page is now Axe clean, with many issues that were fixed applying to the other pages as well. Will continue working through the various sub-pages until the entire site is free of accessibility issues

**Comment by TheBryceIsRight** [comment]
All accessibility issues are now fixed, with the exception of the data vis page. It seems Nivo had one fault after all, which was no A11y support. After reverse engineering some things only 3 issues remain, with 2 of them able to be fixed with an aria-labelledBy tag.
